packages/components/tests/dummy/app/routes/components/table.js

```diff
@@ -5,7 +5,7 @@ export default class ComponentsTableRoute extends Route {
   async model() {
     let response = await fetch('/api/folk.json');
     let { data } = await response.json();
-    const STATES = ['default', 'hover', 'active'];
+    const STATES = ['active', 'default', 'hover', 'focus'];
```

Comment:
Probably intentional, but why have the `active` as first element? to test the border radius?
Reason I am asking is that usually the order we use is `'default', 'hover', 'active', 'focus'`

```suggestion
    const STATES = ['default', 'hover', 'active', 'focus'];
```

Reply:
Is there a reason for an incorrect order? Since it's not the order that the browser recognizes the pseudo states(hover, focus, active), I made them alpha-sorted. I reach for patterns automatically, I suppose.

Reply:
> Is there a reason for an incorrect order
These are the order of the interaction (`default` → `hover` → `active`) plus `focus`